src/app/components/app/AppCalendar.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <Calendar
+    v-slot="{ date }"
+    v-model:placeholder="placeholder"
+    v-bind="forwarded"
+    :class="cn('rounded-md border p-3', props.class)"
+    :locale
+  >
+    <CalendarHeading class="flex w-full items-center justify-between gap-2">
+      <Select
+        :default-value="placeholder?.month.toString()"
+        @update:model-value="
+          (v: unknown) => {
+            if (!v || !placeholder) return
+            if (Number(v) === placeholder?.month) return
+            placeholder = placeholder.set({
+              month: Number(v),
+            })
+          }
+        "
+      >
+        <SelectTrigger aria-label="Select month" class="w-[60%]">
+          <SelectValue placeholder="Select month" />
```

Comment:
Untranslated strings, width could be set to a non-harcoded value maybe or just the style from figma can be implemented